File: Sportics.Api/Entities/Match/MatchInning.cs
namespace Sportics.Api.Entities.Match;

public class MatchInning
{
    public int Id { get; set; }
    public int InnningNo { get; set; }
    public int TeamId { get; set; }
    public int RunsScored { get; set; }
    public int WicketsFallen { get; set; }
    public int MatchId { get; set; }
    public Match? Match { get; set; }
    public ICollection<InningPlayerStats>? InningPlayerStats { get; set; }
}

File: Client/src/app/models/Match/matchInning.ts
import { InningPlayerStats } from './inningPlayerStats';

export interface MatchInning {
  id: number;
  innningNo: number;
  teamId: number;
  runsScored: number;
  wicketsFallen: number;
  matchId: number;
  inningPlayerStats: InningPlayerStats[];
}
